ui/app/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
This is the best I can do:

http://softwareengineering.stackexchange.com/questions/40561/is-bsd-license-compatible-with-apache

Elm itself is under a BSD license, but that doesn't force any sort of license on the code written in the language.

This is technically licensed to me, but I would be happy to open a pull request and change this over to the apache license, and to change ownership to prometheus. Let me know.